### PR TITLE
#159965472 Integrate coveralls code coverage

### PR DESCRIPTION
**Pull request by Derrickkip**
#### What does this PR do?
Integrates coveralls code climate with the badge in the readme

#### Description of tasks to be completed
This PR adds coveralls badge to the readme to ensure we are properly informed how well our code base is tested which in turn provides quality assurance for the product

#### How should this be manually tested?
once the PR is merged the badge in the readme should display the code coverage

#### Relevant PT story
[#159965472](https://www.pivotaltracker.com/story/show/159965472)

### CODE REVIEW

**Review by Quantum-35 — COMMENTED** [review]
Your badge is pointing to master, instead of develop

**Comment by Mutuba** [comment]
Looks good to me!

**Comment by DerKip** [comment]
looks good, coveralls is now pointing to develop

**Review by Teddykavooh — COMMENTED** [review]
Check on your line 5 to 6 linting.

**Comment by Derrickkip** [comment]
@Teddykavooh Please elaborate on the linting

**Comment by Teddykavooh** [comment]
@Derrickkip Sorry, I mean the spacing, have a consistent one. Either of one line or two lines.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-leagueOfLegends/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-leagueOfLegends?branch=develop)
```
Comment (WNjihia) [inline README.md:1]: branch=develop
Good decision 👍

**Review by WNjihia — APPROVED** [review]

**Pull request merged**